.circleci/run_evals_config.yml

```diff
@@ -26,12 +37,14 @@ workflows:
       - run-braintrust-evals:
           context:
             - ai-llm-eval-examples # Replace this with your context name
+            - slack-notification-access-token # Replace this with your context name where SLACK_ACCESS_TOKEN is stored
```

Comment:
Good idea...We'll update the docs accordingly for this option. I think it'll help the user.